plugins/admin/pkg/command/autoscaling/update.go

```diff
@@ -38,8 +40,8 @@ var (
 func NewAutoscalingUpdateCommand(p *pkg.AdminParams) *cobra.Command {
 	AutoscalingUpdateCommand := &cobra.Command{
 		Use:   "update",
-		Short: "update autoscaling config",
-		Long: `update autoscaling config provided by Knative Pod Autoscaler (KPA)
+		Short: "Update autoscaling config",
+		Long: `Update autoscaling config provided by Knative Pod Autoscaler (KPA)
```

Comment:
I kind of expected the Long description to have an example...

Reply:
Yes, we do have examples, there's a new line here

```
For example:
# To enable scale-to-zero
kn admin autoscaling update --scale-to-zero
# To disable scale-to-zero
kn admin autoscaling update --no-scale-to-zero
```